docs/_docs/examples.md

```diff
@@ -28,7 +28,7 @@ key3:
 
 (there is an [issue](YamlDecoder) with deriving YamlEncoder instance for Option datatype hence `YamlDecoder` instead `YamlCodec`)  
 
-```scala sc:compile
```

Comment:
So this is no longer checked in the docs?

Reply:
No, can't be temporarily. Scaladoc has an issue with that.

Reply:
Ok, it seems it's just a scaladox issues. @lbialy please raise it upstream